dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -493,7 +493,7 @@
   <Target Name="ComputeXamlGeneratedCompileInputs" />
   <ImportGroup Label="ExtensionTargets">
     <Import Project="..\..\packages\Microsoft.Gsl.0.1.2.1\build\native\Microsoft.Gsl.targets" Condition="Exists('..\..\packages\Microsoft.Gsl.0.1.2.1\build\native\Microsoft.Gsl.targets')" />
-    <Import Project="..\..\packages\MUXCustomBuildTasks.1.0.37\build\native\MUXCustomBuildTasks.targets" Condition="Exists('..\..\packages\MUXCustomBuildTasks.1.0.37\build\native\MUXCustomBuildTasks.targets')" />
+    <Import Project="..\..\packages\MUXCustomBuildTasks.1.0.38\build\native\MUXCustomBuildTasks.targets" Condition="Exists('..\..\packages\MUXCustomBuildTasks.1.0.38\build\native\MUXCustomBuildTasks.targets')" />
```

Comment:
You might want to move all these versions into a common props file, this seems like it would be a bit complicated to modify.

Reply:
This is what Visual Studio does to manage the nuget package versions if you use the built in UI.

We could probably avoid using that UI and instead do something similar to what we do for WindowsTargetPlatformVersion, to allow the version number to be stored in a central place.

Reply:
Yeah, unfortunately MSBuild doesn't yet have the packages.config as the "truth" for vcxproj. I don't know why. :(